internal/pkg/manifest/lb_web_svc.go

```diff
@@ -85,7 +147,7 @@ func newDefaultLoadBalancedWebService() *LoadBalancedWebService {
 		LoadBalancedWebServiceConfig: LoadBalancedWebServiceConfig{
 			ImageConfig: ServiceImageWithPort{},
 			RoutingRule: RoutingRule{
-				HealthCheckPath: aws.String("/"),
+				HealthCheck: HealthCheckArgsOrString{},
```

Comment:
dumb question: do we need to do the following for the path to render in the manifest by default?

```go
HealthCheckArgsOrString{
   HealthCheckPath: aws.String("/"),
}
```

Reply:
Oh, good call; thank you.